frontend/src/components/SupportedAppTitle.tsx

```diff
@@ -10,18 +10,34 @@ type SupportedAppTitleProps = {
 };
 
 const SupportedAppTitle: React.FC<SupportedAppTitleProps> = ({ odhApp, showProvider = false }) => {
+  const { title, icon } = React.useMemo(() => {
+    const split = odhApp.spec.displayName?.split(' ');
+
+    if (isRedHatSupported(odhApp)) {
+      return {
+        title: `${split.slice(0, -1).join(' ')} `,
+        icon: (
+          <span style={{ whiteSpace: 'nowrap' }}>
+            <span style={{ verticalAlign: 'text-bottom' }}>{split.pop()}</span>
```

Comment:
```suggestion
            <span style={{ verticalAlign: 'text-bottom' }}>{split[split.length - 1]}</span>
```
Small nit -- pop mutates it, so if we ever wanted to use it after this in the future, it would be awkward. It's a nit because I don't think it's a scenario here and neither code is hard to understand what it is doing.